src/App/App.tsx

```diff
@@ -5,7 +5,7 @@ import Layout from './Layout';
 // import Dashboard from './Dashboard';
 import TechnicalDashboard from './TechnicalDashboard';
 import Transfers from './Transfers';
-import ConnectionWizard from './ConnectionWizard';
+// import ConnectionWizard from './ConnectionWizard';
```

Comment:
If you comment out code, please put a comment indicating the reason.

Reply:
i've removed commented out code